epytope/EpitopePrediction/ANN.py

```diff
@@ -888,6 +889,151 @@ def revert_allele_repr(self, name):
,"HLA-C*16:135","HLA-C*16:136","HLA-C*16:137","HLA-C*16:138","HLA-C*16:139","HLA-C*16:14","HLA-C*16:140","HLA-C*16:141","HLA-C*16:142","HLA-C*16:143","HLA-C*16:144","HLA-C*16:145","HLA-C*16:146","HLA-C*16:147","HLA-C*16:148","HLA-C*16:149","HLA-C*16:15","HLA-C*16:150","HLA-C*16:151","HLA-C*16:152","HLA-C*16:153","HLA-C*16:154","HLA-C*16:155","HLA-C*16:156","HLA-C*16:157","HLA-C*16:158","HLA-C*16:159","HLA-C*16:160","HLA-C*16:161","HLA-C*16:162","HLA-C*16:163","HLA-C*16:164","HLA-C*16:17","HLA-C*16:18","HLA-C*16:19","HLA-C*16:20","HLA-C*16:21","HLA-C*16:22","HLA-C*16:23","HLA-C*16:24","HLA-C*16:25","HLA-C*16:26","HLA-C*16:27","HLA-C*16:28","HLA-C*16:29","HLA-C*16:31","HLA-C*16:32","HLA-C*16:33","HLA-C*16:34","HLA-C*16:35","HLA-C*16:36","HLA-C*16:37","HLA-C*16:38","HLA-C*16:39","HLA-C*16:40","HLA-C*16:41","HLA-C*16:42","HLA-C*16:43","HLA-C*16:44","HLA-C*16:45","HLA-C*16:46","HLA-C*16:47","HLA-C*16:48","HLA-C*16:49","HLA-C*16:50","HLA-C*16:51","HLA-C*16:52","HLA-C*16:53","HLA-C*16:54","HLA-C*16:55","HLA-C*16:56","HLA-C*16:57","HLA-C*16:58","HLA-C*16:59","HLA-C*16:60","HLA-C*16:61","HLA-C*16:62","HLA-C*16:63","HLA-C*16:64","HLA-C*16:65","HLA-C*16:66","HLA-C*16:67","HLA-C*16:68","HLA-C*16:69","HLA-C*16:70","HLA-C*16:71","HLA-C*16:72","HLA-C*16:73","HLA-C*16:74","HLA-C*16:75","HLA-C*16:76","HLA-C*16:78","HLA-C*16:79","HLA-C*16:80","HLA-C*16:81","HLA-C*16:82","HLA-C*16:83","HLA-C*16:84","HLA-C*16:85","HLA-C*16:86","HLA-C*16:87","HLA-C*16:88","HLA-C*16:90","HLA-C*16:91","HLA-C*16:92","HLA-C*16:93","HLA-C*16:94","HLA-C*16:95","HLA-C*16:96","HLA-C*16:97","HLA-C*16:98","HLA-C*16:99","HLA-C*17:01","HLA-C*17:02","HLA-C*17:03","HLA-C*17:04","HLA-C*17:05","HLA-C*17:06","HLA-C*17:07","HLA-C*17:08","HLA-C*17:09","HLA-C*17:10","HLA-C*17:11","HLA-C*17:12","HLA-C*17:13","HLA-C*17:14","HLA-C*17:15","HLA-C*17:16","HLA-C*17:17","HLA-C*17:18","HLA-C*17:19","HLA-C*17:20","HLA-C*17:21","HLA-C*17:22","HLA-C*17:23","HLA-C*17:24","HLA-C*17:25","HLA-C*17:26","HLA-C*17:28","HLA-C*17:29","HLA-C*17:30","HLA-C*17:31","HLA-C*17:32","HLA-C*17:33","HLA-C*17:34","HLA-C*17:35","HLA-C*17:36","HLA-C*17:37","HLA-C*17:38","HLA-C*17:39","HLA-C*17:40","HLA-C*17:41","HLA-C*17:42","HLA-C*17:43","HLA-C*17:44","HLA-C*17:45","HLA-C*17:46","HLA-C*17:47","HLA-C*17:48","HLA-C*17:49","HLA-C*17:50","HLA-C*17:51","HLA-C*18:01","HLA-C*18:02","HLA-C*18:03","HLA-C*18:04","HLA-C*18:05","HLA-C*18:06","HLA-C*18:08","HLA-C*18:09","HLA-C*18:10","HLA-C*18:11","HLA-C*18:12","HLA-C*18:13","HLA-C*18:14","HLA-E*01:01","HLA-E*01:03","HLA-E*01:04","HLA-E*01:05","HLA-E*01:06","HLA-E*01:07","HLA-E*01:09","HLA-E*01:10","HLA-E*01:11","HLA-E*01:12","HLA-E*01:13","HLA-E*01:14","HLA-F*01:01","HLA-F*01:02","HLA-F*01:03","HLA-F*01:04","HLA-F*01:05","HLA-F*01:06","HLA-G*01:01","HLA-G*01:02","HLA-G*01:03","HLA-G*01:04","HLA-G*01:06","HLA-G*01:07","HLA-G*01:08","HLA-G*01:09","HLA-G*01:10","HLA-G*01:11","HLA-G*01:12","HLA-G*01:14","HLA-G*01:15","HLA-G*01:16","HLA-G*01:17","HLA-G*01:18","HLA-G*01:19","HLA-G*01:20","HLA-G*01:22"])
```

Comment:
would it make sense to read those from a separate file to be able to read the rest of the code in a nicer way here?